lib/ht_sip_validator/validation/image/count.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module HathiTrust::Validation::Image
+  class Count < HathiTrust::Validation::Base
+
+    def perform_validation
+      image_files = HathiTrust::Validation::Image.image_files(@sip.files)
+      
+      if image_files.count != text_files.count
```

Comment:
Sounds good to me. Some institutions may want to have a way to disable the warning for image count > text count, but we can worry about that later.